src/muqsit/chestshop/Category.php

```diff
@@ -92,6 +87,10 @@ public function send(Player $player, int $page = 1, bool $send = true) : int{
 			foreach($contents as $slot => $item){
 				$inventory->setItem($slot, $item, false);
 			}
+
+            $inventory->setItem(48, Button::get(Button::TURN_LEFT, $this->getRealName()));
```

Comment:
About these, I think it's far better (in terms of perf) to push these into the `$contents` array. Something like this:
```
$contents[48] = Button::get(Button::TURN_LEFT, $this->getRealName());
```
considering the frequency of this being called.